sources/packages/api/src/services/disbursement-schedule-service/disbursement-schedule-service.ts

```diff
@@ -496,4 +532,32 @@ export class DisbursementScheduleService extends RecordDataModelService<Disburse
     firstCOEQuery.orderBy("outstandingCOE.disbursementDate").limit(1);
     return firstCOEQuery.getOne();
   }
+
+  /**
+   **Transformation to convert the data table column name to database column name
+   **Any changes to the data object (e.g data table) in presentation layer must be adjusted here.
+   * @param sortField
+   * @param sortOrder
+   * @returns OrderByCondition
+   */
+  private transformToEntitySortField(
```

Comment:
there is a sort mapper file, 
![image](https://user-images.githubusercontent.com/77353155/153684888-84dd3b65-be2b-4dab-add4-5b975090f060.png)

Reply:
As discussed, the scope of the sort-mapper is just within service. So I prefer private method to achieve it.